app/models/untagged_animal_assessment.rb

```diff
@@ -48,12 +48,20 @@ class UntaggedAnimalAssessment < ApplicationRecord
   )
   validates :length, numericality: true
 
+  def self.create_from_csv_data(attrs)
+    new(attrs)
+  end
+
   # Cohort is translated to shl_case_number to compute stats.
   # Here we need to transfer it back to be able to store value in the DB.
   def shl_case_number=(value)
     self.cohort = value
   end
 
+  def shl_case_number
+    self.cohort
```

Comment:
With `cohort` as the column in the database _and_ in the CSV, this seemed a little cleaner to support referring to an SHL case number when dealing with instances of this model instead of doing translation and retranslation in the import.